### PR TITLE
Bump litd to version `v0.12.4-alpha` 

### PR DESCRIPTION
**Pull request by ViktorTigerstrom**
This PR bumps the litd version to `v0.12.4-alpha`.

This PR also bumps loop to `v0.28.0-beta`, faraday to `v0.2.13-alpha` and lightning-node-connect to `v0.3.1-alpha` and adds the new included protos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -151,6 +152,7 @@ The following table shows the supported combinations:
 
 | LiT               | LND          | Loop         | Faraday       | Pool         | Taproot Assets |
 |-------------------|--------------|--------------|---------------|--------------|----------------|
+| **v0.12.4-alpha** | v0.17.4-beta | v0.28.0-beta | v0.2.11-alpha | v0.6.4-beta  | v0.3.3-alpha   |
```
Comment (guggero) [inline README.md:155]: Faraday version needs to be bumped here as well.

**Review by guggero — APPROVED** [review]
LGTM, pending the version update in the `README.md`.

File: README.md (hunk 1 above)
Comment (ViktorTigerstrom) [reply README.md]: Ah, good catch, thanks!

**Comment by ViktorTigerstrom** [comment]
Tested the loop & pool flow through TW. Have also tested autopilot with a local & staging version of the autopilot server.

**Review by jamaljsr — COMMENTED** [review]
tACK LGTM 🚀

Tested Loop & Pool in the `litd` UI as well as Loop, Pool, Autoloop, and AutoFees in Terminal Web. Everything works great!

**Review by ellemouton — APPROVED** [review]
utACK, LGTM - so I assume we will do another soon with LND 0.18?

**Comment by ViktorTigerstrom** [comment]
> utACK, LGTM - so I assume we will do another soon with LND 0.18?

Yes exactly :)! We wanted this one out to get the new Loop version into terminal as it contains some nice fixes & new features.

**Pull request merged**